qa/smoke-test-reindex-with-all-modules/src/test/resources/rest-api-spec/test/update_by_query/10_script.yml

```diff
@@ -337,6 +337,6 @@
         body:
           script:
             lang: painless
-            source: if (ctx._source.user == "kimchy") {ctx.op = "update"} else {ctx.op = "junk"}
```

Comment:
why is this relevant?

Reply:
because the doc order changes and then we get `update` instead of `junk` and that is invalid too.